File: BandlabInterview/app/src/main/java/com/example/interview/MainActivity.java
package com.example.interview;

import android.app.FragmentManager;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.example.interview.network.commands.GetInitialVideoPageCommand;
import com.example.interview.videopage.VideoPageFragment;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        FragmentManager fm = getFragmentManager();
        fm.beginTransaction()
                .replace(R.id.container, new VideoPageFragment())
                .commit();

//        new GetInitialVideoPageCommand()
//                .start(this, null);
    }
}

File: ExpenseTracker/app/src/main/java/com/example/expensetracker/MainActivity.kt
package com.example.expensetracker

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import com.google.android.gms.auth.api.signin.GoogleSignIn
import com.google.android.gms.auth.api.signin.GoogleSignInClient
import com.google.android.gms.auth.api.signin.GoogleSignInOptions
import com.google.android.gms.drive.Drive
import java.util.concurrent.Executors

class MainActivity : AppCompatActivity() {

    lateinit var client: GoogleSignInClient

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val executor = Executors.newSingleThreadExecutor()
        executor.submit(Runnable() {
            client = buildGoogleSignInClient()
        })
    }

    private fun buildGoogleSignInClient(): GoogleSignInClient {
        val signInOptions = GoogleSignInOptions.Builder(GoogleSignInOptions.DEFAULT_SIGN_IN)
                .requestScopes(Drive.SCOPE_FILE)
                .build()
        return GoogleSignIn.getClient(this, signInOptions)
    }

}
